cli_reference/developer_cli_odo/installing-odo.adoc

```diff
@@ -8,7 +8,9 @@ toc::[]
 
 // The following section describes how to install `{odo-title}` on different platforms using the CLI or the Visual Studio Code (VS Code) IDE.
 
-You can install the `{odo-title}` CLI on Linux, Windows, or macOS by downloading a binary. You can also install the OpenShift VS Code extension, which uses both the `{odo-title}` and the `oc` binaries to interact with your OpenShift Container Platform cluster. For {op-system-base-full}, you can install the `{odo-title}` CLI as an RPM.
+You can install the `{odo-title}` CLI on Linux, Windows, or macOS by downloading a binary. 
```

Comment:
This is an out-of-scope topic, but you might want to consider having separate attributes for the product title and the command. 
For example, https://docs.openshift.com/container-platform/4.11/cli_reference/developer_cli_odo/understanding-odo.html refers to the product as 
```
Red Hat OpenShift Developer CLI (`odo`)
``` 
I also see many instances of the literal command like this throughout the docs. For example:
```
Required for `odo` CLI.
```
Whatever you choose, you might want to make it consistent across the docs. Without that consistency, the benefit of having a command attribute is limited.

Reply:
The OpenShift CLI is never referred to with the Red Hat prefix, so to be consistent I guess I should remove that one instance of Red Hat OpenShift Developer CLI and just call it "OpenShift Developer CLI" there.

So the question then is, do I use  "OpenShift Developer CLI (`odo`)" everywhere? I'm not sure, as `odo` is not an acromyn, so not as clear cut as oc - 

Also odo is an upstream project where it is also a Kubernetes developer CLI as well as OpenShift (whereas oc is OpenShift-only) and the acronym hasn't any meaning whatsoever...

So I am tempted to use plain `odo` everywhere, and reword the understanding-odo.html to say:

> `odo`, the developer CLI for OpenShift and Kubernetes....

Thoughts?

Other examples:
Installing {pipelines-title} CLI (tkn) on Linux using an RPM...
The `opm` CLI tool is provided by the Operator Framework for use with t...
You can customize your Knative (`kn`) CLI setup by creat...

Reply:
I will change the title in the updating-odo file as outlined above, but that is in a separate PR.

I will create a new PR to handle the inconsistencies between using `{odo-title}` variable and `odo` across the docs.